syntax = "proto3";

package response_models;

message UpdatePlayerHitpointsResponseModel {
	int32 player_id = 1;
	int32 new_hitpoints = 2;
}